.github/workflows/rebase.yml

```diff
@@ -2,23 +2,64 @@ name: Automatic Rebase
 on:
   issue_comment:
     types: [created]
+  pull_request_target:
+    types: [auto_merge_enabled]
+env:
+  AUTO_REBASE_PERSONAL_ACCESS_TOKEN: ${{ secrets.AUTO_REBASE_PERSONAL_ACCESS_TOKEN }}
 jobs:
-  rebase:
-    name: Rebase
-    if: github.event.issue.pull_request != '' && contains(github.event.comment.body, '/rebase')
+  check_token:
+    if: ${{ (github.event.issue.pull_request != '' &&
```

Comment:
I'm never entirely sure about the precedence of `&&` and `||`. Guessing this adds the comment on PRs that have the `/rebase` comment OR fire the `pull_request_target` event?